pkg/trace/stats/sublayers.go

```diff
@@ -34,6 +39,195 @@ func (v SublayerValue) GoString() string {
 	return v.String()
 }
 
+// activeSpans is a set of active spans. We could use a map of spans for that.
```

Comment:
```suggestion
// activeSpans helps keep track of spans which are actively executing while iterating through
// all timestamps of a trace inside the SublayerCalculator to compute durations. 
```
Let's first explain what `activeSpanSet` is before mentioning alternatives.